YukonVaccinationVerifier/YukonVaccinationVerifier/Extensions & utilities/UIControl+AddAction.swift

```diff
@@ -0,0 +1,23 @@
+//
+//  UIControl+AddAction.swift
+//  YukonVaccinationVerifier
+//
+//  Created by Mohamed Fawzy on 16/11/2021.
+//
+
+import UIKit
+
+extension UIControl {
+    
+    func addAction(for controlEvents: UIControl.Event = .touchUpInside, _ closure: @escaping()->()) {
```

Comment:
Nice!
Does this cause any sort of memory leak?

Reply:
I guess it won't cause